toolchain/sem_ir/function.h

```diff
@@ -109,10 +75,11 @@ struct Function : public EntityWithParamsBase,
       -> TypeId;
 
   // Returns information about how the function returns its return value.
-  auto GetReturnInfo(const File& file,
-                     SpecificId specific_id = SpecificId::Invalid) const
-      -> ReturnInfo {
-    return ReturnInfo::ForType(file, GetDeclaredReturnType(file, specific_id));
+  auto GetReturnTypeInfo(const File& file,
+                         SpecificId specific_id = SpecificId::Invalid) const
+      -> ReturnTypeInfo {
+    return ReturnTypeInfo::ForType(file,
+                                   GetDeclaredReturnType(file, specific_id));
```

Comment:
FWIW, if it worked, maybe ReturnTypeInfo::ForFunction instead?

Reply:
Done.